Create ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md (#1858)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-15-2-fix-log-output-when-running-local-dev-server-wrong-ports-5881.md

---
date: ""

> title: Fix wrong port numbers in log output when running local development server

Reviewer: ```suggestion
title: Correct port numbers shown in log output when running local development server
```

> title: Fix log output when running local dev server (wrong ports) (#5881) [GRAFT][release/cd] (#5983)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix wrong port numbers in log output when running local development server
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57344
version: 1019.15.2
---

> Fix log output when running local dev server (wrong ports) (#5881) [GRAFT][release/cd] (#5983)

Author: The Claude AI suggested to use the following body:
```suggestion
When running a local development server, the log output incorrectly displayed the port numbers as 8080 and 8443. This has been fixed to show the correct port numbers that the local development server is actually using, which are port 80 for HTTP and port 443 for HTTPS. This improves the developer experience by avoiding confusion about which ports are being used.
```

> When running a local development server with the `ng serve` command, the log output incorrectly displays the port numbers if the default port is already used. This has been fixed to show the correct port numbers that the local development server is actually using. This improves the developer experience by avoiding confusion about which ports are being used.

Reviewer: ```suggestion
When running a local development server with the `ng serve` command, the log output incorrectly displayed the port numbers if the default port was already used. This has been fixed to show the correct port numbers that the local development server is actually using. This improves the developer experience by avoiding confusion about which ports are being used.
```
